flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -249,7 +252,7 @@ public void testKeyedListStateMigration() throws Exception {
                         new TestType.V2TestTypeSerializer()));
     }
 
-    @Test
+    @TestTemplate
     public void testKeyedListStateSerializerReconfiguration() throws Exception {
```

Comment:
```suggestion
    void testKeyedListStateSerializerReconfiguration() throws Exception {
```
Could be package private